docs/_plugins/shortcodes/repoStatus.cjs

```diff
@@ -251,9 +251,12 @@ function repoStatusTable() {
 <tr>
 <td data-label="Name">
   <a href="/elements/${listItem.name}">${listItem.name}</a>
-  ${listItem.overallStatus !== 'Available' ? `<rh-tag color="${STATUS_LEGEND[listItem.overallStatus].color}" variant="${STATUS_LEGEND[listItem.overallStatus].variant}">
-    ${listItem.overallStatus}${STATUS_LEGEND[listItem.overallStatus].icon}
-  </rh-tag>` : ''}
+  ${listItem.overallStatus !== 'Available' ? `
+  <span>
+    <rh-tag color="${STATUS_LEGEND[listItem.overallStatus].color}" variant="${STATUS_LEGEND[listItem.overallStatus].variant}">
+      ${listItem.overallStatus}${STATUS_LEGEND[listItem.overallStatus].icon}
+    </rh-tag>
+  </span>` : ''}
```

Comment:
Should we instead have the span wrap both the `<a>` and the `<rh-tag>` that way it stays on the same line in the mobile view instead of dropping down a line? 

Current:

![image](https://github.com/RedHat-UX/red-hat-design-system/assets/13666/b9ab5fff-6843-4554-aea3-31b48ff42964)

Span wrapping both `<a>` and `<rh-tag>`

![Screenshot 2024-05-06 at 4 35 15 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/13666/8aa2fd35-2df4-4cc8-9df3-deb57f6003b7)

Reply:
@zeroedin thanks for pointing that out, i've pushed a commit for the same